### ISSUE TITLE
Automated a11y tests fail on website homepage

### ISSUE THREAD

**Issue by jps** [newscorp-ghfb/NewsKit#795]
# Description
Whilst working on splitting the a11y tests up into separate files I noticed that the homepage test was missing, when added it failed. instead of making the PR even bigger, the test generation has been commented out in `scripts/generate-a11y-tests-web.ts` 

![image](https://user-images.githubusercontent.com/350953/228815042-9a53293e-367e-4e4b-ae7d-40d62602cfbb.png)

 

# Tasks/Steps

- [ ] reenable the test generation
- [ ] fix the a11y violations (we can ignore the whole component hero showcase section)

# Acceptance Criteria

All a11y tests should pass
